### ISSUE TITLE
Tweet example freeze after moving cursor

### ISSUE THREAD

**Issue by ouchxp** [facebook/draft-js#411]
Type `#@a` in tweet example, move cursor to the left most and then press right, the browser will freeze.
Can reproduce this issue on both Chrome and Safari. Firefox does not have this problem.

**Comment by steviesama** [comment]
I tried to reproduce this in Chrome on [heroku](http://draftjs-examples.herokuapp.com/) and worked fine.

**Comment by ouchxp** [comment]
@steviesama 
The html file I've been testing is directly cloned from this repo, with all the stylesheets and scripts included.
I went to http://draftjs-examples.herokuapp.com/ and it freeze as well. CPU usage is 100% in the activity monitor. My Chrome version is `Version 50.0.2661.102 (64-bit)` OS is `OS X EL Capitan 10.11.4 (15E65)`, also tried on `Windows 8.1` with latest Chrome, both in regular mode and Incognito mode (disabled all extensions).
I tried turn on profiling, but since it freeze the window, the profiler never get stop, I couldn't know what is it doing, and couldn't provide much more useful information regarding this issue.
